src/template.yml

```diff
@@ -271,7 +271,7 @@ Resources:
           S3_BUCKET_NAME: !Ref BootstrapTemplatesBucket
           TERMINATION_PROTECTION: false
           MASTER_ACCOUNT_ID: !Ref AWS::AccountId
-          ADF_VERSION: 3.0.3
+          ADF_VERSION: 3.0.4
```

Comment:
One of the ADF_VERSION environment variables was accidentally changed to `c` with the release of 3.0.3.
It is shown on line 255, but Github doesn't allow me to mark it in this PR directly..., so I am adding it here instead :)

Could you get that updated to 3.0.4 as well?

Reply:
fixed! thanks!